t/fork.t

```diff
@@ -12,10 +12,14 @@ BEGIN {
      $Config::Config{useithreads} and
      $Config::Config{ccflags} =~ /-DPERL_IMPLICIT_SYS/
     );
-  if ( $can_fork ) {
+  if ( $can_fork && !(($^O eq 'MSWin32') && $Devel::Cover::VERSION) ) {
     print "1..8\n";
   } else {
-    print "1..0 # Skip No fork available\n";
+    if ( !$Devel::Cover::VERSION ) {
```

Comment:
This logic looks inverted - shouldn't it be `if ($Devel::Cover::VERSION)` ?

Otherwise, this looks great.

Reply:
Yes, that's a mistake. I'll correct it.

Reply:
🛠 * fixed